### PR TITLE
Adding function to shoplist danny

### PR DESCRIPTION
**Pull request by DNiev**

### CODE REVIEW

**Review by tracilevine — APPROVED** [review]

**Review by jpcas — APPROVED** [review]

File: client/package.json [hunk 1]
```diff
@@ -17,8 +17,12 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
     "react-share": "^4.3.1",
+
```
Comment (ErnieAtLYD) [inline client/package.json:20]: Listen, _I_ personally don't put blank spaces in my `package.json` and maybe a future boss will be super OCD. If it works, then hey.

File: client/src/components/BikeShopCard.js [hunk 2]
```diff
@@ -15,10 +15,13 @@ const BikeShopCard = () => {
         </Card.Body>
 
         <Card.Body>
-          <Card.Title>City Bikes</Card.Title>
-          <Card.Text>888 Bike Chain Road, Miami, Florida, 33137</Card.Text>
-          <Card.Text>305-305-3053</Card.Text>
-          <Card.Text>305-305-3053</Card.Text>
+          <Card.Title>{bikeshop?.shopName}</Card.Title>
+          <Card.Text>
+            {bikeshop?.shopContact.street} {bikeshop?.shopContact.city}{' '}
```
Comment (ErnieAtLYD) [inline client/src/components/BikeShopCard.js:20]: Is there ever a chance that any of those street, city, state, zip code, email, or website fields are blank? If so, does it look okay?

File: client/src/pages/Shoplist.js [hunk 3]
```diff
@@ -1,17 +1,63 @@
...
+        </Card>
         <Map />
-        <h3>Bike Shops</h3>
-        <BikeShopCard />
+
+        <Card>
+          <Card.Title className="fltr-txt">Filter By</Card.Title>
+        </Card>
+        {filteredBikeshops?.map((bikeshop) => {
+          console.log(bikeshop);
```
Comment (ErnieAtLYD) [inline client/src/pages/Shoplist.js:53]: If you don't have a need for `console.log()`, kill it.

**Review by ErnieAtLYD — APPROVED** [review]
@DNiev looks alright to me. I'll go ahead and approve your PR but I'm not gonna merge it as I added a couple of thoughts on comments - from there I'll let you be the judge if you want to add more commits addressing those issues or just pushing.

File: client/src/components/BikeShopCard.js (hunk 2 above)
Comment (tracilevine) [reply client/src/components/BikeShopCard.js:20]: email and website but the rest no

**Pull request merged**